src/microbe_stage/Microbe.cs

```diff
@@ -473,7 +473,11 @@ public override void _Ready()
             if (ColonyChildren != null)
             {
                 foreach (var child in ColonyChildren)
+                {
                     AddChild(child);
+                    AddCollisionExceptionWith(child);
```

Comment:
Adding a comment here explaining that this duplicates some of the logic on joining a colony would be excellent.